libs/dh/metering-point/feature-overview/src/graphql/get-metering-point-by-id.graphql

```diff
@@ -84,7 +84,10 @@ query GetMeteringPointById(
       resolution
       product
       productObligation
-      scheduledMeterReadingMonth
+      scheduledMeterReadingDate {
```

Comment:
suggestion: update mock data with the new property

Reply:
Added